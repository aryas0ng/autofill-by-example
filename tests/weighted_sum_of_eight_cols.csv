1,9,0,2,1,7,2,3,112
2,8,5,1,16,3,6,3,201
2,7,4,0,3,6,1,34,358
3,6,3,2,7,5,8,4,185
10,5,2,1,8,11,6,4,210
5,4,1,0,7,2,1,6,118
4,3,0,9,1,1,21,4,236
19,2,2,8,8,5,3,1,160
2,1,1,7,4,4,4,14,219
1,0,0,6,1,4,6,5,
9,4,6,5,9,6,15,1,
7,3,5,4,8,1,3,1,
4,2,4,3,2,6,4,1,
3,1,3,2,6,4,1,1,
12,0,2,1,13,1,1,11,
4,1,1,0,1,4,9,12,
1,0,0,2,7,5,4,16,
26,5,3,1,10,13,2,16,
8,4,2,0,2,5,2,17,
3,3,1,3,3,1,3,2,
3,2,0,2,1,11,7,2,
8,1,4,1,1,15,23,2,
4,0,3,0,1,2,3,20,
6,9,2,5,1,2,8,20,
5,8,1,4,1,2,3,27,
4,7,0,2,17,2,6,4,
3,6,6,1,2,4,7,4,
2,5,5,0,21,43,3,5,
1,4,4,2,26,5,1,54,
0,3,3,1,4,6,8,6,
3,2,2,3,7,9,6,6,
2,1,1,3,8,3,4,7,
1,0,0,4,4,4,4,8,
0,4,3,46,7,2,17,8,
6,3,2,5,9,5,5,8,
5,2,1,6,5,5,11,15,