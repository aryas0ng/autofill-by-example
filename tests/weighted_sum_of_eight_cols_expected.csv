1,9,0,2,1,7,2,3,112
2,8,5,1,16,3,6,3,201
2,7,4,0,3,6,1,34,358
3,6,3,2,7,5,8,4,185
10,5,2,1,8,11,6,4,210
5,4,1,0,7,2,1,6,118
4,3,0,9,1,1,21,4,236
19,2,2,8,8,5,3,1,160
2,1,1,7,4,4,4,14,219
1,0,0,6,1,4,6,5,136
9,4,6,5,9,6,15,1,249
7,3,5,4,8,1,3,1,119
4,2,4,3,2,6,4,1,114
3,1,3,2,6,4,1,1,91
12,0,2,1,13,1,1,11,188
4,1,1,0,1,4,9,12,197
1,0,0,2,7,5,4,16,230
26,5,3,1,10,13,2,16,319
8,4,2,0,2,5,2,17,212
3,3,1,3,3,1,3,2,82
3,2,0,2,1,11,7,2,151
8,1,4,1,1,15,23,2,298
4,0,3,0,1,2,3,20,211
6,9,2,5,1,2,8,20,283
5,8,1,4,1,2,3,27,294
4,7,0,2,17,2,6,4,197
3,6,6,1,2,4,7,4,152
2,5,5,0,21,43,3,5,451
1,4,4,2,26,5,1,54,628
0,3,3,1,4,6,8,6,179
3,2,2,3,7,9,6,6,204
2,1,1,3,8,3,4,7,161
1,0,0,4,4,4,4,8,153
0,4,3,46,7,2,17,8,431
6,3,2,5,9,5,5,8,212
5,2,1,6,5,5,11,15,288